modules/wlroots/hm.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+
+with config.stylix.colors;
+
+let
+  slideshow = pkgs.writeShellApplication {
+    name = "slideshow";
+    runtimeInputs = [ pkgs.swww ];
+    text = ''
+      images=(${builtins.toString config.stylix.wallpaper.images})
+      while true; do
+        swww img ''${images[ $RANDOM % ''${#images[@]} ]}
+        sleep ${builtins.toString config.stylix.wallpaper.delay}
+      done
+    '';
+  };
+
+  swww = {
+    Unit = {
+      Description = "Wallpaper daemon";
+      Documentation = [ "https://github.com/Horus645/swww#readme" ];
+      PartOf = [ "graphical-session.target" ];
+    };
+    Service = {
+      Type = "notify";
+      NotifyAccess = "all";
+      Environment = [ "PATH=${pkgs.swww}/bin" ];
+      ExecStart = "${pkgs.swww}/bin/swww init --no-daemon";
+      ExecStop = "${pkgs.swww}/bin/swww kill";
```

Comment:
Is `PATH` required in `swww.Service.Environment` when `swww.Service.ExecStart` and `swww.Service.ExecStop` already use the absolute `/nix/store/<swww>/bin/sww` path?

Reply:
I'm not sure whether that syntax is understood by systemd - `ExecStart` isn't read like a normal shell command.

Reply:
Maybe we should just keep `PATH` in `Environment`. Feel free to resolve this discussion.